fsspec-proxy/pyproject.toml

```diff
@@ -30,7 +30,16 @@ dependencies = [
 dynamic = ["version", "urls", "keywords"]
 
 [project.optional-dependencies]
-test = ["anaconda-cloud-storage", "s3fs"]
+s3 = [
+    "s3fs"
+]
+anaconda = [
+    "anaconda-cloud-storage"
+]
+all = [
+    "s3fs",
+    "anaconda-cloud-storage"
```

Comment:
I think you can make these relative to the other groups
```suggestion
    ".[s3]",
    ".[anaconda]"
```
so that if the other groups change, we don't have to edit in two places.

Reply:
Thanks!

Reply:
Unfortunately, we got a syntax error with this. I like the idea though, so I used the brand new PEP735 dependency groups feature.